rpm/install-repos.inc

```diff
@@ -0,0 +1,13 @@
+[archivematica]
```

Comment:
Can we make this part of script.sh? This way, the script can be used for rpm installs , using only one file.